packages/gatsby/src/utils/__tests__/webpack.config.js

```diff
@@ -101,6 +101,10 @@ describe(`environment variables`, () => {
         expect.stringContaining(`.env.staging`),
         expect.anything()
       )
+      expect(DefinePlugin).toBeCalledTimes(1)
+      expect(DefinePlugin.mock.calls[0][0]).toMatchObject({
+        [`process.env.NODE_ENV`]: JSON.stringify(process.env.NODE_ENV),
+      })
```

Comment:
not blocking NIT: this could be
```
expect(DefinePlugin).toBeCalledWith(expect.objectContaining({
  "process.env.NODE_ENV": JSON.stringify(process.env.NODE_ENV),
})
```
(that allows us to not grab jest internals (`.mock.calls[0][0]`) )